### ISSUE TITLE
Forked Due to Apparent Loss of Maintainer

### ISSUE THREAD

**Issue by c-j-s** [kura/vagrant-bash-completion#41]
The maintainer of this repo appears to have vanished (I tried to e-mail him but his domain has expired), so I've taken over maintenance of this for for the moment and incorporated all the PRs as of this date. You can find the fork at [c-j-s/vagrant-bash-completion](https://github.com/c-j-s/vagrant-bash-completion).

**Comment by brbsix** [comment]
I've also got a fork at [brbsix/vagrant-bash-completion](https://github.com/brbsix/vagrant-bash-completion) with some significant updates.

**Comment by c-j-s** [comment]
Argh. Would have been nice if you'd announced this here before I'd spent an hour forking and pulling in all the other changes and the like. This issue is, of course, this repo is the top hit for "vagrant bash completion" on both Google and DuckDuckGo, soltysh and ddarbyson come in on the top page of each respectively and
you don't appear at all, unfortunately. And keep in mind that forks don't appear in [github searches](https://github.com/search?utf8=%E2%9C%93&q=vagrant-bash-completion&type=) either.

Anyway, now that people know about this, do you want to pull in to your repo the changes I have that you don't (`box prune` and the cygwin/mingw fix, from the looks of it) and then I can kill my fork?

**Comment by brbsix** [comment]
Apologies. I suppose it's not particularly convenient to find, but doesn't my fork show up at the top of https://github.com/kura/vagrant-bash-completion/network?

I just added `box prune` support in https://github.com/brbsix/vagrant-bash-completion/commit/d6f10f0413eb03ecccb364cee5c67575a340935c (I had already implemented it a while ago but didn't commit until now). I'll look over the cygwin/mingw fix in a bit.

**Comment by c-j-s** [comment]
Ah, that's a good (if not obvious) way of searching. Anyway, good stuff. I've just updated and done a quick test of the current head of your master branch and everything seems ok. So I'll use that from this point on and kill my own fork. Thanks for the quick response!

**Comment by yermulnik** [comment]
@c-j-s https://github.com/c-j-s/vagrant-bash-completion throws `404 Page not found`

**Comment by c-j-s** [comment]
@yermulnik: Right, as one might expect if one reads the message immediately above yours. :-) But good point; I've edited the initial message in this thread to point directly to @brbsix's repo.
